Algorithm/QCAlgorithm.cs

```diff
@@ -1792,14 +1792,15 @@ public Option AddOption(Symbol underlying, Resolution? resolution = null, string
         /// <param name="market">The futures market, <seealso cref="Market"/>. Default is value null and looked up using BrokerageModel.DefaultMarkets in <see cref="AddSecurity{T}"/></param>
         /// <param name="fillDataForward">If true, returns the last available data even if none in that timeslice. Default is <value>true</value></param>
         /// <param name="leverage">The requested leverage for this equity. Default is set by <see cref="SecurityInitializer"/></param>
+        /// <param name="extendedMarketHours">Show the after market data as well</param>
```

Comment:
nit - more than `show` I think should be something like `use extended market hours data`